src/generate_wsc.f90

```diff
@@ -123,7 +126,7 @@ subroutine generate_wsc(mol,wsc)
          endif
       end do
    end do
-!$omp enddo
-!$omp endparallel
+   !$omp end do
+   !$omp end parallel
```

Comment:
While not strictly necessary, I think it does not hurt to be extra clear about the deallocation in the parallel region (as in [`tblite`](https://github.com/tblite/tblite/blob/ba9c91e8708b332fad67bcb021c7054b544ef7fb/src/tblite/coulomb/charge/effective.f90#L560-L562)).

```suggestion
   !$omp end do
   deallocate(dist, lattr, trans)
   !$omp end parallel
```